setup/asset_distribute.ts

```diff
@@ -0,0 +1,282 @@
+import {BigNumber} from 'ethers';
+import fs from 'fs-extra';
+import hre from 'hardhat';
+import {DeployFunction} from 'hardhat-deploy/types';
+import inquirer from 'inquirer';
+
+const beep = (num: number) => {
+  for (let i = 0; i < num; i++) {
+    console.log('---------------------------------------------------------');
+  }
+};
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+//require('node-beep');
+
+const readOnly = false;
+const BATCH_SIZE = 50;
+
+let totalGasUsed = BigNumber.from(0);
+
+let promptCounter = 1;
+
+const func: DeployFunction = async function () {
+  const {ethers, getNamedAccounts, network} = hre;
+
+  const gasPriceFromNode = await ethers.provider.getGasPrice();
+  let gasPrice = gasPriceFromNode;
+  if (hre.network.name === 'mainnet') {
+    gasPrice = BigNumber.from('56000000000'); // TODO
```

Comment:
What exactly is the TODO referring to?

Reply:
To the ability to configure gas price when executing the script instead of hardcoding it.